agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -89,6 +88,18 @@ async def add_variant_based_on_image(
     if already_exists:
         raise ValueError("App variant with the same name already exists")
 
+    is_new_app = not await does_app_exist(app_variant.app_name, **kwargs)
+    if is_new_app:
```

Comment:
I understand it's a demo version, but I believe we should add a feature that allows us to create new environments for existing apps—especially those made before this pull request—to fully support the publishing functionality.

Reply:
One solution for this is to create the default environments not here but instead at 
`async def list_environments(app_name: str, **kwargs: dict) -> List[Environment]:`

Basically we modify it so that if the list of environments is empty we call a function `initialize_environements` which creates the environments below

Reply:
Thank you for the suggestion; I'll modify the code accordingly. And thanks to Yoon for pointing it out.